lib/pdf_parser.rb

```diff
@@ -0,0 +1,159 @@
+require 'RMagick'
+require 'grim'
+require 'whatlanguage'
+require 'similar_text'
+
+
+module PdfParser
+
+  def build_text_by_hybrid page_text, img_path
```

Comment:
Apparently not? I didn't even know that was valid syntax. Regardless, I prefer all function definitions to have parenthesis even if the language allows them not to.